resources/providers/azurelib/inventory/provider.go

```diff
@@ -212,3 +203,20 @@ func (p *Provider) runPaginatedQuery(query armresourcegraph.QueryRequest) ([]Azu
 
 	return resourceAssets, nil
 }
+
+func (p *Provider) getAssetFromData(data map[string]any) AzureAsset {
+	subId := getString(data, "subscriptionId")
+	properties, _ := data["properties"].(map[string]any)
```

Comment:
Should we have error handling if type assertions fail here?

Reply:
Not much we can do if these are missing since that would either mean the specific asset doesn't have "properties" (I don't know if that's possible) or the output was malformed. Since the code was just moved around, I'd argue to keep the code as is for now and decide on what to do for these cases (also for `getString` which ignores missing values / wrong types).

The code is safe from panics.